src/app/search/search-filters/range-numeric.tsx

```diff
@@ -1,11 +1,12 @@
 import { tl } from 'app/i18next-t';
 import { getItemKillTrackerInfo, getItemYear } from 'app/utils/item-utils';
+import { D2CalculatedSeason, D2SeasonInfo } from 'data/d2/d2-season-info';
 import { FilterDefinition } from '../filter-types';
 import { generateSuggestionsForFilter } from '../search-utils';
 
-const rangeStringRegex = /^([<=>]{0,2})(\d+)$/;
+const rangeStringRegex = /^([<=>]{0,2})(\d+|hardcap|softcap)$/;
 
-export function rangeStringToComparator(rangeString?: string) {
+export function rangeStringToComparator(rangeString?: string, powerKeywords = false) {
```

Comment:
i'd rather do this as another pre-processor instead of adding weird flags & secondary effects into a rather nice single purpose function like `rangeStringToComparator`. see for example:
https://github.com/DestinyItemManager/DIM/blob/master/src/app/search/search-filters/range-overload.tsx#L137
and
https://github.com/DestinyItemManager/DIM/blob/master/src/app/search/search-filters/range-overload.tsx#L89

Reply:
Ah, that should be nicer, yeah